azure-devops/global/abstracts/clone-and-move-plugin-steps.yml

```diff
@@ -13,22 +13,10 @@ parameters:
     default: ''
 
 steps:
-  - task: NodeTool@0
-    inputs:
-      versionSpec: '18.19.0'
-
-  - script: npm install --global yarn
-
-  - script: |
-      git clone --depth 1 --branch ${{ parameters.TAG_NAME }} https://github.com/opensearch-project/OpenSearch-Dashboards.git $(Agent.TempDirectory)/OpenSearch-Dashboards
-    displayName: 'Clone OpenSearch Dashboards repository'
-
-  - task: CopyFiles@2
-    inputs:
-      SourceFolder: '$(Build.Repository.LocalPath)'
-      Contents: '**'
-      TargetFolder: '$(Agent.TempDirectory)/OpenSearch-Dashboards/plugins/${{ parameters.PLUGIN_NAME }}'
-    displayName: 'Copy Plugin to Plugins folder'
+  - template: ./template-clone-and-move-plugin.yml
+    parameters:
```

Comment:
Can you rename this file to be "execute-command-opensearch-dashboards.yaml" please?